views/interface.js

```diff
@@ -0,0 +1,65 @@
+function getPostData() {
+    fetch("https://chitter-backend-api-v2.herokuapp.com/peeps").then(response => {
```

Comment:
Cool use of ES6 syntax - I really need to get comfortable with this, I keep avoiding it